README.md

```diff
@@ -13,19 +13,25 @@ By default there is a "default" image for layouts. But the idea is to customize
 ## Image convention
 
 * The size of image should be 366 x 150 or bigger 732 x 300.
-* They should be named based on the flexible key (`push_2_light`) with no underscores but dashes (`push-2-light.png`).
+* They should be named based on the flexible key (`push_2_light`) with no underscores but dashes (`push-2-light.[extension]`).
+* The following extensions will be resolved in that order: `.jpg`, `.jpeg`, `.png` and `.gif`.
+* The following directories will be resolved in that order: plugin folder, child theme folder, theme folder.
```

Comment:
Should be moved into the "Where images should be situated"

Reply:
Ok, I will move it.